login.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link href="https://fonts.googleapis.com/css?family=Cantarell:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/pokestyles.css">
+    <title>Pokédex</title>
+</head>
+<body>
+<header>
+    <img src="img/pokeball.svg">
+    <h1>POKÈDEX</h1>
+</header>
+<main>
+    <form>
+        <label class="email" for="email" aria-label="email"></label>
```

Comment:
Should say "email" inside label, as not all screen readers use aria-label.

Reply:
fixed in f353331

Reply:
This has now removed accessibility.

Reply:
changed in c5d0917